apps/backend/user-service/src/test/resources/secret.yml

```
@@ -0,0 +1,13 @@
+  datasource:
+    url: jdbc:postgresql://localhost:5432/oakcan
+    username: postgres
+    password: postgres
+  security:
+    jwt:
+      secret: alongrandomstringthatneedstobechangedmaybethisislongenough
+      expiration: 86400000
+    refresh:
+      expiration: 604800000
+  bcrypt:
+    strength: 12
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)